binary/test/Test/Cardano/Binary/Bi.hs

```diff
@@ -16,69 +16,66 @@ import qualified Hedgehog.Gen as Gen
 import qualified Hedgehog.Range as Range
```

Comment:
Is `Test.Cardano.Binary.Bi` still a good name for this module? Shouldn't be renamed to something like `Test.Cardano.Binary.RoundTrip`?